Sources/Utils/RequestModelBeautifier.swift

```diff
@@ -71,6 +70,33 @@ class RequestModelBeautifier: NSObject {
         txt.append("------------------------------------------------------------------------\n\n\n\n")
         return txt
     }
+
+    static func curlExport(request: RequestModel) -> String {
+        func serializeHeader(dictionary: [String: String]) -> String {
+            var result = ""
+
+            dictionary.keys.sorted().forEach { key in
+                guard let value = dictionary[key] else { return }
+                result += " -H '\(key.lowercased()): \(value)'"
+            }
+
+            return result
+        }
+
+        let curl = "curl"
+        let headers = request.headers.map(serializeHeader) ?? ""
+
+        let serializedBody = request.httpBody.map { body -> String in
+            return " --data-binary '\(String(decoding: body, as: UTF8.self))'"
+        }
+
+        var body = "\(serializedBody ?? "")"
+        if request.method.lowercased() != "post" {
```

Comment:
POST requests in cURL do not need `-X POST` since it already contains `--data-binary`. Furthermore, JSON payload outputted by Charles used `--data-binary`, although [extensive documentation for cURL](https://ec.haxx.se/http-post.html) showed that `-d '{json}' -H 'Content-Type: application/json'` was enough.

As mentioned in the PR's description, I tried to follow Charles as closely as possible

Reply:
Tbh I would still add `-X POST`. I didn't know this parameter is optional for POST and when I get some `cURL` I very often look what HTTPMethod it is.

I guess the reason why we want to have this `cURL` is to share with others. In that case I think it's better to have it (unless it's a big "code style" issue in cURL)

Reply:
As mentioned, during development I chose to follow Charles' approach as much as possible since their export to cURL works well. This is simply a consequence of doing so.

Nevertheless we can still keep `-X POST`; personally I prefer having it there.

Reply:
Done.